build.sbt

```diff
@@ -9,10 +9,18 @@ val circeVersion = "0.14.14"
 val awsVersion = "2.31.60"
 val zioVersion = "2.1.19"
 
+lazy val scalafmtSettings = Seq(
+  scalafmtFilter.withRank(KeyRanks.Invisible) := "diff-dirty",
```

Comment:
this means that when running the tests it will check the formatting of any files that have changed